recipe/meta.yaml

```diff
@@ -28,6 +28,7 @@ test:
     - cf_xarray
   requires:
     - pytest
+    - pkg_resources
```

Comment:
`pkg_resources` is part of setuptools and we need that as a run dependency b/c cf_xarray imports it. BTW, if you run `grayskull pypi --strict-conda-forge cf_xarray` you can regenerate this recipe based only on the package metadata, anything that does not work after that should be fixed upstream. See https://github.com/xarray-contrib/cf-xarray/pull/269